app/components/on-site-notifications/notification-card.js

```diff
@@ -295,6 +303,101 @@ export default Component.extend(
 			});
 		},
 
+		getMessageWallPostBody(model) {
+			let wallOwner = model.get('metadata') && model.get('metadata.username');
+		
+			if (!wallOwner) {
+				wallOwner = this.getMessageWallOwner(model.get('uri'));
+			}
+		
+			const isOwnWall = wallOwner === this.usernameMarkup;
+			const args = {
+				postTitle: this.postTitleMarkup,
```

Comment:
This should be set to `model.get('title')`. There is a `this.titleMarkup` computed value which is for bolding the title, but that's not what we want. For the wall notifications, the title is bolded in the localized strings where necessary. And `this.postTitleMarkup` is undefined.

Reply:
Yep. Makes sense. Updated.